frontend/public/config.yml

```
@@ -266,20 +182,59 @@ collections:
         name: 'thank_you'
         file: 'frontend/src/content/pages/thank_you.md'
         fields:
+          - { label: 'Thank you page title', name: 'thank_you_title', widget: 'string' }
+          - { label: 'Thank you text', name: 'body', widget: 'markdown' }
+          - { label: 'Thank you image', name: 'thank_you_image', widget: 'image' }
+
+      - label: 'LARP Rules'
+        name: 'larp_rules'
+        file: 'frontend/src/content/pages/larp-rules.md'
+        fields:
+          - { label: 'LARP rules page title', name: 'larp_rules_title', widget: 'string' }
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)